portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -372,6 +372,27 @@ definition {
 			value1 = "${validationErrorMessage}");
 	}
 
+	macro editValidationTargetArgument {
+		Pause(locator1 = "1000");
```

Comment:
@Jamersonze, usually, we try to not use `Pause` on our tests, instead we first try to use something like:

```
WaitForElementPresent(locator1 = "Form#DESCRIPTION_FIELD");

Click(locator1 = "Form#DESCRIPTION_FIELD");
```

Can you please give it a try for the `FormFields#VALIDATION_ARGUMENT`?

Reply:
The wait wasn't necessary.